keystorage/src/main/java/tech/pegasys/web3signer/keystorage/hashicorp/config/ConnectionParameters.java

```diff
@@ -58,4 +59,49 @@ public long getTimeoutMilliseconds() {
   public URI getVaultURI() {
     return vaultURI;
   }
+
+  public HttpClient.Version getHttpProtocolVersion() {
+    return httpProtocolVersion;
+  }
+
+  public static final class Builder {
```

Comment:
nice improvement 👍

Reply:
Yes, I realised that this class is used in our tests as well and we were forced to set values that were not required by the specific tests. Builder patterns neatly allows the defaults to be set.